nengo_loihi/tests/test_loihi_api.py

```diff
@@ -1,23 +1,24 @@
 import numpy as np
+import pytest
 
 from nengo_loihi.loihi_api import overflow_signed
 
 
-def test_overflow_signed():
-    for b in (8, 16, 17, 23):
-        x = np.arange(-2**(b-2), 2**(b+2), dtype=np.int32)
+@pytest.mark.parametrize("b", (8, 16, 17, 23))
```

Comment:
Made this a parameter, as it makes things slightly easier to debug if one of these values starts failing.